integration/test/basic/basic_test.go

```diff
@@ -27,39 +32,55 @@ func TestChartInstalled(t *testing.T) {
 		RestConfig:      f.RestConfig(),
 		TillerNamespace: "giantswarm",
 	}
-	helmClient, err := helmclient.New(c)
+
+	gsHelmClient, err := helmclient.New(c)
 	if err != nil {
 		t.Fatalf("could not create helmClient %v", err)
 	}
 
-	err = f.InstallResource("chart-operator-resource", templates.ChartOperatorResourceValues, ":stable")
+	// Test Creation
+	log.Printf("creating %q", cr)
+	err = f.InstallResource(cr, templates.ChartOperatorResourceValues, ":stable")
+	if err != nil {
+		t.Fatalf("could not install %q %v", cr, err)
+	}
+
+	err = waitForReleaseStatus(gsHelmClient, testRelease, "DEPLOYED")
+	if err != nil {
+		t.Fatalf("could not get release status of %q %v", testRelease, err)
+	}
+	log.Printf("%q succesfully deployed", testRelease)
+
+	// Test Deletion
+	log.Printf("deleting %q", cr)
+	err = helmClient.DeleteRelease(cr)
```

Comment:
Note: Here we are using the regular helmClient, since `chart-operator-resources` are installed with the normal tiller.